kore/src/Kore/Log/DebugUnifyBottom.hs

```diff
@@ -55,6 +55,7 @@ instance Pretty DebugUnifyBottom where
 
 instance Entry DebugUnifyBottom where
     entrySeverity _ = Debug
+    oneLineDoc _ = mempty
```

Comment:
Do you think it would be useful to default to `show` if there is no detailed information available? Or is it better to omit the message? This also affects other instances that default to `mempty`.

If we stay with `mempty`, does it produce an empty line or no line at all?

Reply:
I used `mempty` because that's what [Max suggested](https://github.com/kframework/kore/issues/2781#issuecomment-893590259). The one line log looks like this:
```
kore-exec: [361804] Debug (DebugUnifyBottom):
```
Perhaps it is confusing to have nothing after the colon mark, but using `oneLineDoc = Pretty.pretty . show` would print everything in Haskell syntax, like
```
kore-exec: [357145] Debug (DebugUnifyBottom): DebugUnifyBottom {info = "...", first = TermLike {getTermLike = TermAttributes {termSort = SortActualSort (SortActual {sortActualName ...
```
Do you think `DebugUnifyBottom _` would be better in this case, and similarly for the others cases that use `mempty`?

Reply:
Thanks for the clarification. Yes, I think having something after the colon mark would be a good thing for the reason you mentioned. The type-name seems to be a reasonable choice; even if it doesn't provide the user with extra knowledge, it can avoid confusion.